src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/ProviderBase/DbConnectionInternal.cs

```diff
@@ -203,6 +203,14 @@ virtual protected bool ReadyToPrepareTransaction
             }
         }
 
+        virtual internal bool IsAccessTokenExpired
```

Comment:
Is there any cases that IsAccessTokenExpired returns true? I have seen we check this property in one of the conditions in DbConnectionPool.cs : 
` if(null != obj && obj.IsAccessTokenExpired)`